templates/sios-datakeeper.template.yaml

```diff
@@ -1,12 +1,11 @@
 ---
 AWSTemplateFormatVersion: '2010-09-09'
-Description: 'This template deploys a two or three node Windows Server Failover Cluster (WSFC)
+Description: This template deploys a two- or three-node Windows Server Failover Cluster (WSFC)
   with SQL Server 2017 and SIOS DataKeeper replication software. This template is
```

Comment:
@davmayd Same question as above (re: "Server 2017" here).

Reply:
@MarciaRieferJohnston it defaults to 2017 but allows other versions (https://github.com/aws-quickstart/quickstart-sios-datakeeper/blob/main/templates/sios-datakeeper.template.yaml#L384), if you'd prefer to omit the SQL server version I have no objection

Reply:
Thanks, Dave. Done.